curator-recipes/src/test/java/org/apache/curator/framework/recipes/nodes/TestPersistentNode.java

```diff
@@ -136,4 +138,52 @@ public void testQuickCloseNodeExists() throws Exception
             CloseableUtils.closeQuietly(client);
         }
     }
+
+    @Test
+    public void testInterruption() throws Exception
```

Comment:
I don't understand what this is testing. What does it matter how many times a thread is interrupted? I feel like this is an academic exercise. Is there a real problem here?